### PR TITLE
Fix text deselect

### LINKED ISSUE #819: Text pasted has white background on white
When I paste any text in to Hyper.app the pasted text shows white background on white:

screen shot 2016-10-07 at 1 38 37 pm

I have no plugins.

### PR DESCRIPTION
**Pull request by ppot**
Close #819

### CODE REVIEW

**Comment by ppot** [comment]
I can't review myself. @matheuss @albinekb @rauchg

**Comment by naturalethic** [comment]
This is still not fixed on MacOS at least.  First of all, e.code is `ShiftLeft`, `ControlLeft`, etc.  But even if I hack it to check these, and ensure that `selection.clear` is not being called, the selection is still being cleared by something else.

**Comment by ppot** [comment]
On MacOS this work perfectly. This is not released yet. If you still have a problem. feel free to make a gif. With your specific use case.

**Comment by naturalethic** [comment]
I was hacking on it in dev mode using the repo master branch.  A production build to dist doesn't work for me, it won't open a new window.  Anyhoo, I'm fine waiting for the next official release.  Thanks again.

File: lib/hterm.js [hunk 1]
```diff
@@ -213,20 +213,22 @@ hterm.Keyboard.prototype.onKeyDown_ = function (e) {
 
   if (e.altKey || e.metaKey || isAccelerator(e)) {
     // hterm shouldn't consume a hyper accelerator
+    // // Was the hyperCaret removed for selectAll
+    if (e.key === 'v' && !this.terminal.cursorNode_.contains(this.hyperCaret)) {
```
Comment (matheuss) [inline lib/hterm.js]: @ppot why this is here? :thinking:

**Comment by Yliaho** [comment]
spans that have white background color are still obfuscated... I had to google this issue, sorry if this isn't the right place to moan about this.  
  
This fixes *all* of the white-against-white issues:  
  
`termCSS: 'span[style*="background-color: rgb(255, 255, 255);"] {color: black!important;}'`

**Comment by bmherold** [comment]
> This fixes all of the white-against-white issues:
> termCSS: 'span[style*="background-color: rgb(255, 255, 255);"] {color: black!important;}'

with "backgroundColor: 'rgba(0,0,0,0.6)'," I'm still seeing white on white, even with the termCSS setting above.

**Comment by ppot** [comment]
If you have issue. We recommend creating new one and linking it to the closed one.

**Pull request merged**